index.html

```diff
@@ -76,6 +76,15 @@
       postProcess: [window.respecVc.createVcExamples],
       license: "w3c-software-doc",
       xref: true,
+      localBiblio:  {
+        "SD-JWT": {
+          title:    "Selective Disclosure for JWTs (SD-JWT)",
+          href:     "https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-04",
```

Comment:
```suggestion
          href:     "https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-05",
```
will be published today.